src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$card-text-hover-color: #34568b;
+$header-link-color: #00acdc;
+$transist-durations: 300ms;
```

Comment:
Can I reuse any of these colors for other elements or are these colors only for card hover and header links?